bot/cogs/watchchannels/watchchannel.py

```diff
@@ -0,0 +1,353 @@
+        user_id = msg.author.id
+
+        guild = self.bot.get_guild(GuildConfig.id)
+        actor = guild.get_member(self.watched_users[user_id]['actor'])
+        actor = actor.display_name if actor else self.watched_users[user_id]['actor']
+
+        inserted_at = self.watched_users[user_id]['inserted_at']
+        time_delta = self._get_time_delta(inserted_at)
+
+        reason = self.watched_users[user_id]['reason']
+
+        embed = Embed(description=f"{msg.author.mention} in [#{msg.channel.name}]({msg.jump_url})")
+        embed.set_footer(text=f"Added {time_delta} by {actor} | Reason: {reason}")
+
+        await self.webhook_send(embed=embed, username=msg.author.display_name, avatar_url=msg.author.avatar_url)
+
+    async def list_watched_users(self, ctx: Context, update_cache: bool = True) -> None:
+        """
+        Gives an overview of the watched user list for this channel.
+
+        The optional kwarg `update_cache` specifies whether the cache should
+        be refreshed by polling the API.
+        """
+        if update_cache:
+            if not await self.fetch_user_cache():
+                await ctx.send(f":x: Failed to update {self.__class__.__name__} user cache, serving from cache")
+                update_cache = False
+
+        lines = []
+        for user_id, user_data in self.watched_users.items():
+            inserted_at = user_data['inserted_at']
+            time_delta = self._get_time_delta(inserted_at)
+            lines.append(f"• <@{user_id}> (added {time_delta})")
+
+        lines = lines or ("There's nothing here yet.",)
+        embed = Embed(
+            title=f"{self.__class__.__name__} watched users ({'updated' if update_cache else 'cached'})",
+            color=Color.blue()
+        )
+        await LinePaginator.paginate(lines, ctx, embed, empty=False)
+
+    @staticmethod
+    def _get_time_delta(time_string: str) -> str:
+        """Returns the time in human-readable time delta format"""
+        date_time = datetime.datetime.strptime(
+            time_string,
+            "%Y-%m-%dT%H:%M:%S.%fZ"
+        ).replace(tzinfo=None)
+        time_delta = time_since(date_time, precision="minutes", max_units=1)
+
+        return time_delta
+
+    @staticmethod
+    def _get_human_readable(time_string: str, output_format: str = "%Y-%m-%d %H:%M:%S") -> str:
+        date_time = datetime.datetime.strptime(
+            time_string,
+            "%Y-%m-%dT%H:%M:%S.%fZ"
+        ).replace(tzinfo=None)
+        return date_time.strftime(output_format)
+
+    def _remove_user(self, user_id: int) -> None:
+        """Removes user from the WatchChannel"""
+        self.watched_users.pop(user_id, None)
+        self.message_queue.pop(user_id, None)
+        self.consumption_queue.pop(user_id, None)
+
+    def cog_unload(self) -> None:
+        """Takes care of unloading the cog and canceling the consumption task."""
+        self.log.trace(f"Unloading {self.__class__._name__} cog")
```

Comment:
Remove the name from the message. It was missing an underscore by the way.

```suggestion
        self.log.trace(f"Unloading the cog")
```